aspnet/signalr/overview/performance/using-signalr-performance-counters-in-an-azure-web-role/samples/sample10.cs

```diff
@@ -15,38 +15,36 @@ public static string createSignalRCategoryName()
         {
             if (RoleEnvironment.IsEmulated)
```

Comment:
As mentioned above, these APIs are deprecated, so we don't need this code anymore (since we register for all SignalR apps in the wadcfgx file).